velox/core/SimpleFunctionMetadata.h

```diff
@@ -26,8 +26,8 @@
 #include "velox/type/Type.h"
 #include "velox/type/Variant.h"
 
-inline constexpr char kPrecisionVariable[] = "a_precision";
-inline constexpr char kScaleVariable[] = "a_scale";
+static inline constexpr char kPrecisionVariable[] = "a_precision";
```

Comment:
Hi Karteek, These "static" key words again triggered the linter warning we seen before. May I try removing the static if this doesn't break things?

```
Using namespace-level `static` in a .h file is near-always a bad idea: for simple types, static is implied; for aggregate types, static linkage means that a copy of the aggregate will appear in every compilation unit that includes the header; and for functions, static means the function will have a separate body generated in each compilation unit including the header.
```

Reply:
@kagamiori  I removed the static keyword and just made it inline like before. Thanks for trying.